src/providers/moduleCalls.ts

```diff
@@ -83,7 +83,18 @@ export class ModuleCallsDataProvider implements vscode.TreeDataProvider<ModuleCa
         }
       }),
       vscode.window.onDidChangeActiveTextEditor(async (event: vscode.TextEditor | undefined) => {
-        if (event && getActiveTextEditor()) {
+        const activeEditor = getActiveTextEditor();
+
+        const document = activeEditor?.document;
+        if (document === undefined) {
+          return;
+        }
+
+        if (!isTerraformFile(document)) {
```

Comment:
I think we can pass  `activeEditor?.document` directly into this function since it can handle an undefined value. This allows us to remove the check for undefined above.

```ts
 if (!isTerraformFile(activeEditor?.document)) {
```

This also applied to the next check and the two in `src/providers/moduleProviders.ts`